tests/inputFields.spec.ts

```diff
@@ -110,21 +92,21 @@ test('Pet type name is required validation', async ({page}) => {
 
   /* 5. Add the assertion for the "Name is required" message below the input field */
   // Expecting to see the message to fill in the pet type name
-  expect(await page.locator('div').filter({ hasText: 'Name' }).nth(-1).textContent()).toEqual('Name is required')
+  await expect(page.locator('div').filter({ hasText: 'Name' }).nth(-1)).toHaveText('Name is required')
```

Comment:
This locator would look more compact: `page.locator('div', {hasText: 'Name'}).last()`
When you use method filter() it does exactly the same thing as a second argument for locator() method.
filter() method is useful when you need to make additional filtering of the output, like multiple filters applied to the locator.